{% extends 'catalog/base.html' %}

{% block content %}

<div class="information">
   <h1>Study identification</h1>
   <p class="small">&nbsp;</p> 
   <p>Published epigenome-wide association studies are identified through periodic PubMed searches. The search terms are "epigenome-wide" OR "epigenome wide" OR "EWAS" OR "genome-wide AND methylation" OR "genome wide AND methylation". There are also restrictions placed on the year of publication (i.e. studies after 2010) and whether the study is in human samples.</p>
   <p class="small">&nbsp;</p> 
   <h1>Study eligibility</h1>
   <h2>Study inclusion</h2>      
   <p>Studies are eligible for inclusion in the EWAS Catalog if:</p>
   <ul>
      <li>Include at least 100,000 CpG sites in the analysis.</li>
      <li>Have a sample size of at least 100 individuals.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h2>Study exclusion</h2>      
   <p>Studies were not included in the EWAS Catalog if:</p>
   <ul>
      <li>The DNA methylation data studied was not genome-wide.</li>
      <li>The study does not include any new EWAS data.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h2>CpG inclusion</h2>      
   <p>Association results with CpGs are eligible for inclusion in the EWAS Catalog if:</p>
   <ul>
      <li>The association has <i>p</i> &lt 1&times10<sup>-4</sup>.</li>
      <li>The analysis was performed genome-wide.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h1>Data extraction</h1>  
   <p>The data for the EWAS Catalog are manually extracted from the published literature. The information extracted falls into four categories: study information, analysis information, participants information, and CpG results.</p>
   <p class="small">&nbsp;</p>   
   <h2>Study information</h2>        
   <p>The following pieces of study and publication information are extracted:</p>
   <ul>
      <li>Author - the first author of the publication (surname then initials).</li>
      <li>Consortium - the name of the consortium.</li>
      <li>PMID - the PubMedID of the publication.</li>
      <li>Date - the date the paper was published (YY-MM-DD).</li>
      <li>Trait - the name of the trait.</li>
      <li>EFO - the corresponding <a href="http://www.ebi.ac.uk/efo/">ontology term(s)</a> for the trait. </li>
      <li>Analysis - description of the analysis performed.</li>
      <li>Source - the table where the result can be found in the paper.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h2>Analysis information</h2>      
   <p>The following pieces of information on the analysis are extracted:</p>
   <ul>
      <li>Outcome - the outcome of the analysis.</li>
      <li>Exposure - the exposure of the analysis.</li>
      <li>Covariates - the covariates adjusted for in the analysis.</li>
      <li>Outcome_Units - the units of the outcome.</li>
      <li>Exposure_Units - the units of the exposure.</li>
      <li>Methylation_Array - the array used to measure the methylation.</li>
      <li>Tissue - the tissue in which the methylation was measured.</li>
      <li>Further_Details - any other relevant details of the analysis.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h2>Participants information</h2>      
   <p>The following pieces of information on the participants are extracted:</p>
   <ul>
      <li>N - the total number of participants used in the analysis.</li>
      <li>N_Cohorts - the total number of cohorts used in the analysis.</li>
      <li>Categories - the total number of individuals in each category (e.g. 200 smokers, 150 never smokers).</li>
      <li>Age - the mean age of the participants in years.</li>
      <li>N_Males - the total number of males used in the analysis.</li>
      <li>N_Females - the total number of females used in the analysis.</li>
      <li>N_EUR - the total number of European participants.</li>
      <li>N_EAS - the total number of East Asian participants.</li>
      <li>N_SAS - the total number of South Asian participants.</li>
      <li>N_AFR - the total number of African participants.</li>
      <li>N_AMR - the total number of Admixed American participants.</li>
      <li>N_OTH - the total number of participants of non-EUR, non-EAS, non-SAS, non-AFR, non-AMR ancestry.</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h2>CpG results</h2>      
   <p>The following information on the CpG associations are extracted:</p>
   <ul>
      <li>CpG - the CpG site.</li>
      <li>Beta - the effect estimate.</li>
      <li>SE - the standard error of beta.</li>
      <li>P - <i>p</i>-value.</li>
      <li>Details - any additional details on the analysis (e.g. sub-trait).</li>
   </ul>
   <p class="small">&nbsp;</p> 
   <h1>Query options</h1>  
   <p>The EWAS Catalog has four query options:</p>
   <ul>
      <li>CpG - this presents all results with the CpG site. A CpG ID or a hg19 chromosome:position can be queried.</li>
      <li>Gene - this presents all CpG results within the gene.</li>
      <li>Region - this presents all CpG results within the region.</li>
      <li>Trait - this presents all the CpG results with a trait. This query uses <a href="http://www.ebi.ac.uk/spot/zooma/">ZOOMA</a> and selects results based on their EFO terms.</li>
   </ul>
   <p>See example queries under the search box.</p>
   <p class="small">&nbsp;</p> 
   <h1>Output</h1>  
   <p>On the screen a table of results is presented with a subset of columns (or variables) to browse. The full dataset with all variables for the query is available to download. This file is a tab-deliminated tsv file with the same variables as in the <a href="{% url 'catalog_download' %}#variables">downloadable catalog</a>.</p>    
</div>

{% endblock %}
